tests/sanity/test/explorer-apis/test-api-datasets-v2.js

```diff
@@ -98,7 +98,7 @@ describe('test explorer server datasets api v2',async function() {
 
   });
 
-  it(`should be able to get content of data set ${TEST_DATASET_NAME} (v2 API)`, async() => {
+  it(`should be able to get content of a data set member ${TEST_DATASET_NAME} (v2 API)`, async() => {
```

Comment:
`should be able to get content of a data set member ${TEST_DATASET_NAME} (v2 API)`
change to
`should be able to get content of a data set member ${TEST_DATASET_NAME}(${TEST_DATASET_MEMBER_NAME}) (v2 API)`

This shows what member content will be retrieved rather than just saying a member of the given dataset.

Reply:
Agreed, I have updated that!